cacti/tests/conftest.py

```diff
@@ -0,0 +1,49 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import os
+from copy import deepcopy
+
+import pytest
+
+from datadog_checks.cacti import CactiCheck
+from datadog_checks.dev import TempDir, WaitFor, docker_run, run_command
+
+from .common import CONTAINER_NAME, E2E_METADATA, HERE, INSTANCE_INTEGRATION, RRD_PATH
+
+
+def set_up_cacti():
+    commands = [
+        ['/sbin/restore'],
+        ['mysql', '-u', 'root', '-e', "flush privileges;"],
+        ['php', '/opt/cacti/lib/poller.php', '--force'],
+    ]
+    for c in commands:
+        command = ['docker', 'exec', CONTAINER_NAME] + c
+        run_command(command, capture=True, check=True)
```

Comment:
Don't we favour using python docker client now ? :)

Reply:
This PR is prior to introducing the docker cli  (which was only merged yesterday :p)